dev-lang/nasm/nasm-2.14.02.recipe

```diff
@@ -42,7 +51,8 @@ TEST_REQUIRES="
 
```

Comment:
Need `defineDebugInfoPackage` since nasm was built with `-g`

Reply:
What do you mean `-g`?

Reply:
If you pay attention to the building process, you'll find that `gcc` was called with `-g` (generate debuginfo) flag.

Reply:
So should we define for `nasm` and `ndisasm` or nasm only?

Reply:
All binaries.